Aplib.Tests/Belief/ListBeliefTests.cs

```diff
@@ -0,0 +1,79 @@
+using Aplib.Core.Belief;
+using System.Collections.Generic;
+
+namespace Aplib.Core.Tests.Belief;
+
+/// <summary>
+/// Unit tests for <see cref="ListBelief{TReference,TObservation}" />.
+/// </summary>
+public class ListBeliefTests
+{
+    /// <summary>
+    /// A constant 'false' method for testing.
+    /// </summary>
+    /// <returns><c>false</c>.</returns>
+    private static bool NeverUpdate() => false;
```

Comment:
Nit: I prefer to just use () => false instead.

Reply:
Like:
```csharp
private const Func<bool> _neverUpdate = () => false;
```
?